01-js-intro-and-jquery/keypoints/1_11/resources/script/script.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Created by MaximilianoDaniel on 11/09/2015.
+ */
+
+var onPageReady = function () {
+    var processRequest;
+    var showError;
+    var showComplete;
+    var listLenght;
+    var listAlbums;
+    var showAlbum;
+
+    showAlbum = function(album){
+
+    };
+    showError = function(xmlHttpRequest,errorText,objectException) {
+        window.alert('error');
+    };
+
+    showComplete = function(xmlHttpRequest,result) {
+        window.alert('complete');
+    };
+
+    processRequest = function(result) {
+        listLenght = result.albums.items.length;
+        listAlbums = result.albums.items;
+
+        window.alert('succes');
+        console.log(result);
+        if(listLenght){
+            $.each(listAlbums, function(index,album){
+                showAlbum(album);
+            })
+        }
+
+    };
+
+
+
+    $.ajax({
+            url:'https://api.spotify.com/v1/search?',
+            type: 'get',
+            dataType: 'json',
+            data: {
+                q: 'Rolling Stones',
+                type: 'album'
+            },
+            success: processRequest,
+            error: showError,
+            complete: showComplete
+        }
+    );
+};
+
+$(document).on('ready', onPageReady);
+
+
+
```

Comment:
This file is quite right, just minor comments, most of them are coding issues. Maybe the console log ( the one to show the album) should be on the showAlbum function, the one that is empty.